### ISSUE TITLE
Not possible to specify default API server port when using `cardano-wallet-launcher`.

### ISSUE THREAD

**Issue by jonathanknowles** [cardano-foundation/cardano-wallet#410]
| Release | Operating System       | Cause |
| ---     | ---                    | ---   |
| next    | Windows & OSX & Linux) | Code |

# Context

The `cardano-wallet-launcher` command makes it easy to start `cardano-http-bridge` and `cardano-wallet server` in a single step.

Once started, a user can then use `cardano-wallet` to issue client commands.

The `cardano-wallet` command is pre-configured with helpful defaults. Indeed, the help text claims that the `--port` argument is optional. If omitted, then port `8090` will be used (either for listening, or for connecting).

However, the current version of `cardano-wallet-launcher` **always** instructs the wallet API server to listen on a **random port**.

This means a user can no longer rely on the default when issuing client commands. (They will **always** have to supply a port.)

This negates the convenience of having a default.

# Steps to Reproduce

In one terminal, issue the following command (without specifying any ports):
```
$ cardano-wallet-launcher
```

In another terminal, issue the following command (without specifying any ports):
```
$ cardano-wallet wallet list
```
## Expected behavior

We see the following result:
```
$ cardano-wallet wallet list
Ok.
[]
```

## Actual behavior
 
We see the following result:
```
$ cardano-wallet wallet list
HttpExceptionRequest Request {
  host                 = "localhost"
  port                 = 8090
  secure               = False
  requestHeaders       = [("Accept","application/json;charset=utf-8,application/json")]
  path                 = "/v2/wallets"
  queryString          = ""
  method               = "GET"
  proxy                = Nothing
  rawBody              = False
  redirectCount        = 10
  responseTimeout      = ResponseTimeoutDefault
  requestVersion       = HTTP/1.1
}
 (ConnectionFailure Network.Socket.connect: <socket: 11>: does not exist (Connection refused))
```

---

# Resolution Plan

# PR

| Number   | Base      |
| ---      | ---       |
| #411        | `develop` |

# QA

**Comment by jonathanknowles** [comment]
Fix available: #411

**Comment by KtorZ** [comment]
As explained in https://github.com/input-output-hk/cardano-wallet/pull/411#issuecomment-502013598, not a bug. The launcher will be merged with the wallet soon (most probably today) and therefore we're trying to avoid any extra work in the launcher code that is already present in the `cardano-wallet` executable.
